controller/helper/process-survey-instances.js

```diff
@@ -12,17 +12,48 @@ const moment = require('moment');
  * @returns {Object} Complience chart data
  */
 function processSurveyInstances (surveys) {
+    const filterSurveyByState = surveys.filter((survey) => {
+        return survey.state === 'completed';
+    });
+
+    const surveyInstanceLength = 0;
+
+    if (filterSurveyByState !== 'undefined' && filterSurveyByState.length > surveyInstanceLength) {
```

Comment:
Ok. In pickDates function, code snippet
"const lastDate = moment(new Date(surveys[0]
        .dateCompleted)
        .toISOString()).utc().format('MM/DD/YYYY HH:mm');
    const missingValue = -1;"
is failing. I will convert it to Array.map() which i think will work.